containers/builder/build.sh

```diff
@@ -18,10 +18,16 @@ done
 
 cd /tmp/project
 
-# Install required dependencies
-npm install
-
-WORKFLOW_PATH=${WORKFLOW_PATH:-"workflows.tsx"}
+# Use yarn or pnpm if there is a yarn.lock or pnpm-lock.yaml
+if [ -f "yarn.lock" ]; then
+  yarn install
+elif [ -f "pnpm-lock.yaml" ]; then
```

Comment:
nit: should we default to preferred tools first in case both exist (ie use pnpm by default)